components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/datasource/QueryExecutor.java

```diff
@@ -425,8 +428,44 @@ public void createRuleTemplatesTable() throws BusinessRulesDatasourceException {
         }
     }
 
-    private PreparedStatement getStatementForInsertingBusinessRule(Connection conn, String businessRuleUUID, byte[] businessRule,
-                                                                   int deploymentStatus, int artifactCount) throws SQLException {
+    public boolean isBusinessRulesTableExist() {
+        Connection conn = null;
```

Comment:
why do we need null assignment?

Reply:
Since these are local variables, they have to be initialized.